gemrb/plugins/GUIScript/PythonConversions.h

```diff
@@ -236,6 +236,7 @@ PyObject* PyString_FromResRef(const ResRef& resRef);
 PyObject* PyString_FromString(const char* s);
 PyObject* PyString_FromStringView(StringView sv);
 PyObject* PyString_FromStringObj(const std::string&);
+PyObject* PyString_FromSystemStringObj(const std::string&);
```

Comment:
what is `PyString_FromStringObj(const std::string&)` used for now? I can only think of `path_t` strings, but that would fall into "system encoding".

Reply:
Apparent remaining users of this method deal with 2DA/IDS files.

Reply:
Then those shouldn't be using `return PyUnicode_Decode(s.c_str(), s.length(), core->TLKEncoding.encoding.c_str(), "strict");`, but probably just utf-8. In fact, nothing should be using TLK encoding in GUIScript.

We should probably just combine these and add a parameter for the encoding.

Reply:
For my German IWD2, there is a `CHARSTR.2DA` file with umlaute encoded in ISO-8559-1. So not sure what else is out there.

Reply:
I'm surprised to hear there is a 2da that uses localized text, but whatever decoding needs to happen should happen elsewhere (like the data importer) and TLK encoding is still probably only relevant to TLKs. I have to assume it's possible for TLK encoding to differ from that of random data files.

We can of course address this later.

Reply:
Btw, that table is currently unused, so it may not even matter (I don't have it here to check and IESDP doesn't either). IDS files should all be fine.